modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -139,34 +139,66 @@
 		}
 	end
 
+	m.elements.linuxGlobals = function(prj)
+		return {
+			-- Common
+			m.projectGuid,
+			m.projectName,
+			m.preferredToolArchitecture,
+			m.toolsVersion,
+
+			-- Linux
+			m.linuxApplicationType
+		}
+	end
+
 	m.elements.globalsCondition = function(prj, cfg)
 		return {
 			m.windowsTargetPlatformVersion,
 			m.xpDeprecationWarning,
 		}
 	end
 
+	m.elements.linuxGlobalsCondition = function(prj, cfg)
+		if cfg.system ~= prj.system then
+			return {
+				m.linuxApplicationType
+			}
+		end
+	end
+
 	function m.globals(prj)
 
 		-- Write out the project-level globals
 		m.propertyGroup(nil, "Globals")
-		p.callArray(m.elements.globals, prj)
+		if prj.system == p.LINUX then
+			p.callArray(m.elements.linuxGlobals, prj)
+		else
+			p.callArray(m.elements.globals, prj)
+		end
```

Comment:
Possibly
```suggestion
		p.callArray(iif(prj.system == p.LINUX, m.elements.linuxGlobals, m.elements.globals), prj)
```
Apply to others `if prj.system == p.LINUX then`.

Reply:
If you don't mind I think the `if then` makes it clearer, if I end up giving the Android module integration a go I'd have to do it anyway

Reply:
Was just a proposal ;-) ok with `if then`.